.github/workflows/ci.yml

```
@@ -53,21 +39,12 @@ jobs:
           node-version: 22.11
           cache: 'npm'
       - run: npm ci
-      - run: echo "${{ vars.ENV_FILE }}" >> .env
-      - run: npm run test
-      - name: Setup Pages
-        uses: actions/configure-pages@v4
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v3
-        with:
-          path: 'reports/html-report'
-      - name: Deploy to GitHub Pages
-        id: deployment
-        uses: actions/deploy-pages@v4
+      - run: npx playwright install
+      - run: npm run codeceptjs
```

или, если конфиг не менять, то тут команду запуска заменить на:
```suggestion
      - run: npm run codeceptjs:headless
```